test/puppetlabs/puppetdb/query/nodes_test.clj

```diff
@@ -109,9 +109,11 @@
                                              [3 "node_c" 3 1]
                                              [4 "node_d" 2 3]
                                              [5 "node_e" 5 2]]]
-      (sql/insert-record :certnames {:certname node})
-      (sql/insert-record :factsets {:certname node :timestamp (to-timestamp (-> facts-age days ago))})
-      (sql/insert-record :catalogs {:id id :hash node :api_version 0 :catalog_version 0 :certname node :timestamp (to-timestamp (minus right-now (-> catalog-age days)))})))
+      (let [factset-timestamp (to-timestamp (-> facts-age days ago))
+            catalog-timestamp (to-timestamp (minus right-now (-> catalog-age days)))]
+        (sql/insert-record :certnames {:certname node})
+        (sql/insert-record :factsets {:certname node :timestamp factset-timestamp :producer_timestamp factset-timestamp})
+        (sql/insert-record :catalogs {:id id :hash node :api_version 0 :catalog_version 0 :certname node :timestamp catalog-timestamp :producer_timestamp catalog-timestamp}))))
```

Comment:
FWIW, since you're already changing the code here (and anywhere else relevant), I'd be tempted to wrap the long lines.